### PR TITLE
Completed prep-css-flexbox exercise. - Brandon Moy

### PR DESCRIPTION
**Pull request by brandon-moy**
prep-css-align-items
prep-justify-content
prep-css-flex-wrap

### CODE REVIEW

File: prep-css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: calc(100% / 3);
```
Comment (thebearingedge) [inline prep-css-flexbox/align-items.html]: Hi Brandon, `flex-basis` goes on flex item elements, so try moving this to a rule set for the `.card-wrapper` class.

File: prep-css-flexbox/flex-wrap.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+<head>
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
```
Comment (thebearingedge) [inline prep-css-flexbox/flex-wrap.html]: Here as well.

File: prep-css-flexbox/justify-content.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+<head>
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      justify-content: center;
+      flex-basis: 25%;
```
Comment (thebearingedge) [inline prep-css-flexbox/justify-content.html]: Here as well.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-css-flexbox/align-items.html (hunk 1 above)
Comment (brandon-moy) [reply prep-css-flexbox/align-items.html]: Corrected
prep-css-align-items-corrected
!

File: prep-css-flexbox/flex-wrap.html (hunk 2 above)
Comment (brandon-moy) [reply prep-css-flexbox/flex-wrap.html]: This one too.
prep-css-flex-wrap-corrected

File: prep-css-flexbox/justify-content.html (hunk 3 above)
Comment (brandon-moy) [reply prep-css-flexbox/justify-content.html]: Thank you Tim :)
prep-css-justify-content-corrected

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**